### PR TITLE
Lesson 1 4

### PR DESCRIPTION
**Pull request by larasodjati**
Moved to-do list into state, added control to "Add to-do" input, added new to-do to list, destructured the props, and prevented blank input from being added to the list.

### CODE REVIEW

File: src/AddTodoForm.js [hunk 1]
```diff
@@ -9,9 +9,11 @@ function AddTodoForm({ onAddTodo }) {
   };
   const handleAddTodo = (event) => {
     event.preventDefault();
-    onAddTodo({ id: Date.now(), title: todoTitle });
-    console.log(todoTitle);
-    setTodoTitle('');
+    if (todoTitle.trim() !== '') {
```
Comment (Atos20) [inline src/AddTodoForm.js:12]: I like what I see here, we want  to make sure we don't save empty string in the todo list

Comment (Atos20) [reply src/AddTodoForm.js:12]: Let's start thinking about where in your code we could start adding these type of validations, 💡

**Comment by Atos20** [comment]
Nice job with this PR, I really enjoyed the way you made your commits. Keep it up!

**Comment by larasodjati** [comment]
> Nice job with this PR, I really enjoyed the way you made your commits. Keep it up!

@Atos20 Thank you so much for the comment and feedback, I really appreciate it!

**Pull request merged**